routes/booking.rb

```diff
@@ -0,0 +1,12 @@
+class App < Sinatra::Base
+  get '/booking/:route_id' do
+    @booking = BookingPresenter.new(params)
+
+    slim :booking
+  end
+
+  post '/booking' do
+    BookingService::CreateOrder.call(params)
```

Comment:
Looks clean & nice :+1: 

Question - what if something goes wrong during order creation and you have to display form errors? Have you anticipated such scenario? Is it even possible for such thing to happen considering your app design?

Also, add empty line between service call & `redirect`

Reply:
Form data is validated on server. If something is wrong, then record isn't created in db. There gonna be a client-side validation, but it's not implemented yet. I believe it'd be enough.